bin/pdagentd.py

```diff
@@ -58,9 +57,9 @@
         )
 
 # Check Python version.
-if sys.version_info[0:2] not in ((2, 6), (2, 7)):
+if sys.version_info.major == 2 and sys.version_info.minor < 7:
```

Comment:
Would this let someone run PDAgent with python version 1.x? 😅 I know that's pretty unlikely, but...

Reply:
It would — but yeah, the one major version we _have_ managed to kick.